EveCM/package.json

```diff
@@ -39,18 +39,19 @@
     "webpack-cli": "^2.1.5"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.1.0",
     "axios": "^0.18.0",
     "babel-polyfill": "^6.26.0",
     "bootstrap": "^4.1.1",
```

Comment:
Does reactstrap use bootstrap as a peer dependency? If so, can you try removing it from the dependency list and see if the app still works?